kafka-ui-react-app/src/components/Nav/ClusterMenuItem/ClusterMenuItem.styled.ts

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
```

Comment:
```suggestion
import { styled } from 'lib/themedStyles';
```
So that we have typization